bitnami/kube-prometheus/templates/exporters/kube-controller-manager/servicemonitor.yaml

```diff
@@ -0,0 +1,42 @@
+{{- if .Values.kubeControllerManager.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "kube-prometheus.fullname" . }}-kube-controller-manager
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: {{ template "kube-prometheus.fullname" . }}-kube-controller-manager
+spec:
+  jobLabel: component
```

Comment:
For example, this is **correct** and the jobLabel should be `component` due to the fact that Kubernetes labels `kube-controller-manager` with:
```yaml
Name:                 kube-controller-manager-gcp-sjrvf
Namespace:            kube-system
Labels:               component=kube-controller-manager
                      tier=control-plane
```

Therefore in this case you want all metrics being collected from this, to be labeled with:
```
{
  job: `kube-controller-manager`
}
```

By default, this is how normally Prometheus works. Of course that this can be extended if needed.